Mathlib/MeasureTheory/Integral/SetIntegral.lean

```diff
@@ -1072,6 +1072,23 @@ theorem Continuous.integral_pos_of_hasCompactSupport_nonneg_nonzero [IsFiniteMea
 
 end OpenPos
 
+section Support
+
```

Comment:
The `MeasurableSpace` can be inferred from the measure: using an implicit argument instead of a typeclass argument makes the lemmas usable in case the measure is defined on another sigma-algebra.

Also, let's put all those in a `variable` line (please remove them from the two lemmas).

```suggestion

variable {M : Type*} [NormedAddCommGroup M] [NormedSpace ℝ M]
  {mX : MeasurableSpace X} {ν : Measure X} {F : X → M}

```